airflow/models.py

```diff
@@ -2010,6 +2130,130 @@ def get(cls, key, session, deserialize_json=False):
         return v
 
 
+class XCom(Base):
+    """
+    Base class for XCom objects.
+    """
+    __tablename__ = "xcom"
+
+    id = Column(Integer, primary_key=True)
+    key = Column(String)
```

Comment:
`sqlalchemy.exc.CompileError: (in table 'xcom', column 'key'): VARCHAR requires a length on dialect mysql`

`String(512)` ?